libs/base/configkeys.h

```diff
@@ -137,4 +137,11 @@
 #define CFG_PINS_PORT_SIZE 210
 #define CFG_BOOTLOADER_PROTECTION 211
 
+#define CFG_LORA_MISO 220
```

Comment:
Better use `CFG_PIN_LORA_MISO`; given that `CFG_PIN_*` stuff is getting close to 100 (which is D0), let's move these to say 1001, 1002, ...

Reply:
moved